src/components/SelectionList/BaseListItem.tsx

```diff
@@ -82,6 +83,7 @@ function BaseListItem<TItem extends ListItem>({
                     onSelectRow(item);
                 }}
                 disabled={isDisabled && !item.isSelected}
+                interactive={isInteractive}
```

Comment:
Thanks @tienifr for the PR. This tests well but I have just one comment here.
Why do we pass `isInteractive` all along the way from  BaseSelectionList--> ListItem --> UserListItem-->BaseListItem? Can we not use `item.isInteractive` instead here?